inginious/frontend/user_manager.py

```diff
@@ -19,6 +19,23 @@
 import pymongo
 from binascii import hexlify
 import os
+import re
+
+
+def sanitize_email(email: str) -> str:
```

Comment:
Isn't it better to set this as a classmethod within the UserManager class?

Reply:
Fixed in a9bbbaa903b5fbb0e1533ac13c416393cf4a5a78.